R-package/configure.ac

```diff
@@ -29,7 +29,7 @@ fi
 if test `uname -s` = "Darwin"
 then
   OPENMP_CXXFLAGS='-Xclang -fopenmp'
-  OPENMP_LIB='-lomp'
+  OPENMP_LIB='-I/opt/homebrew/opt/libomp/include -lomp -L/opt/homebrew/opt/libomp/lib'
```

Comment:
hmm .. hardcoded path might not be preferable. Are we sure that users should be using libomp with R considering rtools contains a complete toolchain.

Reply:
You're right. It is a bad idea to hardcode the path. Homebrew actually uses a different location for Intel Macs and for M1 Macs.

Intel Macs: `/usr/local/opt/`
M1 Macs: `/opt/homebrew/opt/`